Sources/ArcGISToolkit/Components/Popups/PopupView.swift

```diff
@@ -44,9 +44,17 @@ import ArcGIS
 ///
 /// **Behavior**
 ///
+/// As of 200.8, PopupView uses a NavigationStack internally to support browsing utility network
+/// associations. As a result, a PopupView now requires a navigation context isolated from any
```

Comment:
Nitpick. I dropped "now" from the Feature Form version. Take or leave.

```suggestion
/// associations. As a result, a PopupView requires a navigation context isolated from any
```